### PR TITLE
Create skeleton for noisy histograms use case

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: draft-wang-ppm-differential-privacy.md [hunk 1]
```diff
@@ -597,10 +618,11 @@ class Prio3SumVecWithRandomizedResponse:
 > TODO(issue #10): replace `rr_mechanism` once we have concretely defined
 > it in {{rr}}.
 
-### Prio3Histogram with Discrete Gaussian
 
-For Prio3Histogram, we will apply an Aggregator-only DP mechanism,
-that is implemented with discrete Gaussian.
+### Prio3Histogram with Aggregator Noise
+
+> TODO Describe a policy that is compatilbe with Prio3Histogram and that
```
Comment (junyechen1996) [inline draft-wang-ppm-differential-privacy.md]:
```suggestion
> TODO Describe a policy that is compatible with Prio3Histogram and that
```

File: draft-wang-ppm-differential-privacy.md [hunk 2]
```diff
@@ -515,9 +515,30 @@ class DpPolicy:
...
-### Prio3SumVec, `bits = 1`, with Randomized Response Client-DP
+> TODO: Specify integration of a `DpPolicy` into DAP.
+
+# Use Cases
+
+## Histograms
+
+Many applications require aggregating histograms in which each Client submits a
+bit vector with exactly one bit set. We describe two policies for this use
+case: one which targets the OAMC trust model and provides optimal utility; and
```
Comment (junyechen1996) [inline draft-wang-ppm-differential-privacy.md]: I don't think readers will be convinced on optimal utility if we don't provide some numbers, but I'm fine with leaving the text as-is, and we will work on it later on.

**Review by junyechen1996 — APPROVED** [review]

File: draft-wang-ppm-differential-privacy.md [hunk 3]
```diff
@@ -515,9 +515,30 @@ class DpPolicy:
...
+another which targets the more stringent OAOC trust model and provides
+sub-optimal utility.
+
+### Prio3Histogram with Client Noise
```
Comment (wangshan) [inline draft-wang-ppm-differential-privacy.md]:
```suggestion
### Prio3Histogram with Client-DP
```

File: draft-wang-ppm-differential-privacy.md [hunk 4]
```diff
@@ -597,10 +618,11 @@ class Prio3SumVecWithRandomizedResponse:
 > TODO(issue #10): replace `rr_mechanism` once we have concretely defined
 > it in {{rr}}.
 
-### Prio3Histogram with Discrete Gaussian
 
-For Prio3Histogram, we will apply an Aggregator-only DP mechanism,
-that is implemented with discrete Gaussian.
+### Prio3Histogram with Aggregator Noise
```
Comment (wangshan) [inline draft-wang-ppm-differential-privacy.md]:
```suggestion
### Prio3Histogram with Aggregator-DP
```

**Review by wangshan — APPROVED** [review]
I think it's good enough to merge as a skeleton

File: draft-wang-ppm-differential-privacy.md (hunk 2 above)
Comment (cjpatton) [reply draft-wang-ppm-differential-privacy.md]: Good call, I've added an author note with this suggestion. (Author note: `[JC: ...]`)

**Pull request merged**